src/hospital/covid_examinations/models/worker.py

```diff
@@ -0,0 +1,39 @@
+"""Hospital worker model."""
+from django.db import models
+
+from person import Person
+from department import Department
+
+
+class HospitalWorker(models.Model):
+    NURSE = "Nurse"
+    DOCTOR = "Doctor"
+
+    JOBS = tuple(
+        {
+            NURSE: NURSE,
+            DOCTOR: DOCTOR,
+        }.items()
+    )
+
+    job = models.CharField(null=False,
+                           blank=False,
+                           choices=JOBS,
+                           max_length=20)
+
+    department = models.ForeignKey(Department,
+                                   null=False,
+                                   blank=False,
+                                   related_name="workers",
+                                   on_delete=models.CASCADE)
+
+    personal_information = models.OneToOneField(Person,
```

Comment:
I think that foreignkey is better since you dont want to limit yourself to 1 job for a person.
There may be a case in the future when person will have two positions in the hospital